utils/src/bulk-insert-patients.ts

```diff
@@ -0,0 +1,144 @@
+  // Based off the apiKey it will determine the cx to add to the patients.
+  fs.createReadStream("../insert-patients.csv")
+    .pipe(csv())
+    .on("data", async data => {
+      const metriportPatient = mapCSVPatientToMetriportPatient(data);
+      results.push(metriportPatient);
+    })
+    .on("end", async () => {
+      for (const [i, patient] of results.entries()) {
+        try {
+          await sleep(3000);
+          const createdPatient = await metriportAPI.createPatient(patient, facilityId);
+          console.log(i, createdPatient);
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        } catch (error: any) {
+          errors.push({
+            firstName: patient.firstName,
+            lastName: patient.lastName,
+            dob: patient.dob,
+            message: error.message,
+          });
+        }
+      }
+      console.log(errors);
+    });
+}
+
+function sleep(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+const mapCSVPatientToMetriportPatient = (csvPatient: {
+  firstname: string;
+  lastname: string;
+  patient_dob: string;
+  address1: string;
+  city: string;
+  state_name: string;
+  zipcode: string;
+  phone: string;
+  gender: string;
+}): PatientCreate => {
+  return {
+    firstName: csvPatient.firstname,
+    lastName: csvPatient.lastname,
+    dob: csvPatient.patient_dob,
+    genderAtBirth: csvPatient.gender === "Female" ? "F" : "M",
+    address: {
+      ...(csvPatient.address1 ? { addressLine1: csvPatient.address1 } : {}),
+      ...(csvPatient.city ? { city: csvPatient.city } : {}),
+      ...(csvPatient.state_name ? { state: states[csvPatient.state_name] } : {}),
+      zip: csvPatient.zipcode,
+      country: "USA",
+    },
+    personalIdentifiers: [],
+    ...(csvPatient.phone ? { contact: { phone: csvPatient.phone } } : {}),
+  };
+};
+
+function getEnvVar(varName: string): string | undefined {
+  return process.env[varName];
+}
+function getEnvVarOrFail(varName: string): string {
+  const value = getEnvVar(varName);
+  if (!value || value.trim().length < 1) {
+    throw new Error(`Missing ${varName} env var`);
+  }
+  return value;
+}
+
+main();
+
+const states: { [k in string]: USState } = {
+  Arizona: USState.AZ,
+  Alabama: USState.AL,
+  Alaska: USState.AK,
+  Arkansas: USState.AR,
+  California: USState.CA,
+  Colorado: USState.CO,
+  Connecticut: USState.CT,
+  Delaware: USState.DE,
+  Florida: USState.FL,
+  Georgia: USState.GA,
+  Hawaii: USState.HI,
+  Idaho: USState.ID,
+  Illinois: USState.IL,
+  Indiana: USState.IN,
+  Iowa: USState.IA,
+  Kansas: USState.KS,
+  Kentucky: USState.KY,
+  Louisiana: USState.LA,
+  Maine: USState.ME,
+  Maryland: USState.MD,
+  Massachusetts: USState.MA,
+  Michigan: USState.MI,
+  Minnesota: USState.MN,
+  Mississippi: USState.MS,
+  Missouri: USState.MO,
+  Montana: USState.MT,
+  Nebraska: USState.NE,
+  Nevada: USState.NV,
+  "New Hampshire": USState.NH,
```

Comment:
is there a reason a handful of these are in quotes? Does that affect anything?

I'm being nitpicky, I just wanted an excuse to comment on one of your PRs. My company, Welld, is in the process of integrating with your Device API. It's been helpful when we can pop into your repo to see what's going on under the hood - open source is awesome 😎

Reply:
@jrohrbaugh its mostly because keys can't have spaces unless they are within a string. That's great to hear, glad you are finding use out it. We're not bias or anything but we think open source is pretty awesome as well 😎